src/Autosuggest.js

```diff
@@ -74,6 +74,12 @@ class Autosuggest extends Component {
     super();
 
     this.saveInput = this.saveInput.bind(this);
+    this.onDocumentMouseUp = this.onDocumentMouseUp.bind(this);
+  }
+
+  componentDidMount() {
+    global.window.addEventListener('mouseup', this.onDocumentMouseUp, false);
+    global.window.addEventListener('touchend', this.onDocumentMouseUp, false);
```

Comment:
Sorry this is probably a mistake where I was experimenting with using `document` or `window` to see if it had an effect and I didn't change it back.

I just did some reading on it and it seems that for mouse/touch events it makes no difference, I'll change it to bind to `global.document` to be more consistent.